src/NLog/Internal/Fakeables/AppDomainWrapper.cs

```diff
@@ -102,23 +100,19 @@ public AppDomainWrapper(AppDomain appDomain)
         /// <summary>
         /// Process exit event.
         /// </summary>
-        public event EventHandler<EventArgs> ProcessExit;
+        public event EventHandler ProcessExit
```

Comment:
Do you know of this is a breaking change? I think it is (binary breaking change)